can-component.js

```diff
@@ -348,6 +352,18 @@ var Component = Construct.extend(
 
 			// ## Rendering
 
+			// Hook up any <content> with which the component was instantiated
+			var componentContent = componentTagData.content;
```

Comment:
I would probably try to "consolidate" the arguments passed in in a single spot ... that way the arguments are "known" from that part forward.

Reply:
Updated, thanks for the feedback